ROADMAP.md

```diff
@@ -57,7 +57,7 @@ That way you can write your stories how they are best, and preview them how you
 
 We believe in the power of react, and think it's the right choice for a lot of projects.
 But it's up to you and your team to decide your stack.
-Unfortunately if you choose anything other then React or React-Native you can not use storybook.
+Unfortunately if you choose anything other then React or React-Native or Vue or Angular you can not use storybook.
```

Comment:
I think somewhere should be a list of supported frameworks, and we will be able to refer to it.

Reply:
What would be a good place for that?

Reply:
Maybe the main README? Or a separate `FRAMEWORKS.md` ? I also wanted to push this 
https://gist.github.com/igor-dv/d621819b0d772aea0df88608f74e4da3
somewhere.., So maybe to the same place =)